src/manager/msWindowManager.ts

```diff
@@ -728,6 +728,17 @@ export class MsWindowManager extends MsManager {
         ) {
             return false;
         }
+        if (
+            metaWindow.get_role() !== '' &&
+            getSettings('layouts')
+                .get_string('roles-excluded')
+                .split(',')
+                .map((item) => item.trim())
+                .indexOf(metaWindow.get_role()) > -1
+        ) {
```

Comment:
`get_role` is probably a simple getter, but it feels better to bring it out to a variable.
```suggestion
        const windowRole = metaWindow.get_role();
        if (
            windowRole !== '' &&
            getSettings('layouts')
                .get_string('roles-excluded')
                .split(',')
                .map((item) => item.trim())
                .indexOf(windowRole) !== -1
        ) {
```